package.json

```diff
@@ -69,13 +70,14 @@
         "eleventy-plugin-nesting-toc": "^1.3.0",
         "eleventy-plugin-page-assets": "^0.3.0",
         "eleventy-plugin-reading-time": "^0.0.1",
-        "eslint": "^8.19.0",
-        "eslint-config-eslint": "^7.0.0",
-        "eslint-plugin-jsdoc": "^39.2.0",
-        "eslint-plugin-jsx-a11y": "^6.5.1",
+        "eslint": "^8.56.0",
+        "eslint-config-eslint": "^9.0.0",
+        "eslint-plugin-jsdoc": "^48.0.4",
```

Comment:
I think this isn't used anywhere because `eslint-config-eslint` uses its own `eslint-plugin-jsdoc` dependency.